src/platform/configuration/config/config.service.ts

```diff
@@ -90,6 +90,8 @@ export class KonfigService {
           ?.innovationLibrary,
         foundation: this.configService.get(ConfigurationTypes.PLATFORM)
           ?.foundation,
+        contactsupport: this.configService.get(ConfigurationTypes.PLATFORM)
+          ?.contactsupport,
```

Comment:
We need a default setting in `alkemio.yml` `platform.support`

Reply:
sorry, forgot that file, Bobby already noticed. It's pushed